platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformCoordinator.java

```diff
@@ -31,17 +33,23 @@ public class PlatformCoordinator {
     private final InOrderLinkerWiring inOrderLinkerWiring;
     private final LinkedEventIntakeWiring linkedEventIntakeWiring;
     private final EventCreationManagerWiring eventCreationManagerWiring;
+    private final ApplicationTransactionPrehandlerWiring applicationTransactionPrehandlerWiring;
+    private final StateSignatureCollectorWiring stateSignatureCollectorWiring;
 
     /**
      * Constructor
      *
-     * @param internalEventValidatorWiring  the internal event validator wiring
-     * @param eventDeduplicatorWiring       the event deduplicator wiring
-     * @param eventSignatureValidatorWiring the event signature validator wiring
-     * @param orphanBufferWiring            the orphan buffer wiring
-     * @param inOrderLinkerWiring           the in order linker wiring
-     * @param linkedEventIntakeWiring       the linked event intake wiring
-     * @param eventCreationManagerWiring    the event creation manager wiring
+     * @param internalEventValidatorWiring           the internal event validator wiring
+     * @param eventDeduplicatorWiring                the event deduplicator wiring
+     * @param eventSignatureValidatorWiring          the event signature validator wiring
+     * @param orphanBufferWiring                     the orphan buffer wiring
+     * @param inOrderLinkerWiring                    the in order linker wiring
+     * @param linkedEventIntakeWiring                the linked event intake wiring
+     * @param eventCreationManagerWiring             the event creation manager wiring
+     * @param applicationTransactionPrehandlerWiring the application transaction prehandler wiring, null if legacy
+     *                                               prehandle is enabled
+     * @param stateSignatureCollectorWiring      the system transaction prehandler wiring, null if legacy prehandle
```

Comment:
doc mentions this may be null, but param is annotated `@NonNull`

Reply:
fixed javadoc